tools/sggolangcilint/tools.go

```diff
@@ -53,6 +53,9 @@ func Run(ctx context.Context, args ...string) error {
 		if err != nil {
 			return err
 		}
+		if filepath.Dir(path) == sg.FromSageDir() {
+			args = append(args, "--exclude", "(is a global variable|is unused)")
```

Comment:
Is this omitting the output from the terminal? Shouldn't we then turn off those linters instead?

Reply:
Unfortunately we cant combine `--disable-all` and `--disable`.

Reply:
But we control the linter config. We can just remove this from the config right?

https://github.com/einride/sage/blob/200e20f34b6e8a0d1f3819e0647b1f5d064a39b9/tools/sggolangcilint/golangci.yml#L181

Reply:
And the same for the unused linter

Reply:
But I also think those are good linters so I am not sure I understand the reason behind the PR (Or did we discuss reasons for removing them elsewhere?)

Reply:
Thats the thing, we dont want to remove them from the default config. Only for the `.sage` folder, given that vars is common there, and so is unused methods since thats the default for any Sage target

Reply:
I tried excluding deadcode linter in a specific folder, but it didn't take. I think it is because of how golangci-lint is invoked. So I think this is a good solution to the problem

Reply:
Yep, what @Edholm said. We invoke `golangcilint` from the root of the `go.mod`. So excluding by folder work unfortunately

Reply:
Ah I see it now. Bummer.